### PR TITLE
be explicit about dependencies we're pulling in for common-lib

### PR DESCRIPTION
**Pull request by elharo**
@patflynn As discussed yesterday, I'm starting by removing the fileTree rule that pulls in everything.

### CODE REVIEW

**Comment by elharo** [comment]
ping

File: common-lib/build.gradle [hunk 1]
```diff
@@ -7,11 +7,25 @@ repositories {
...
+
+    // todo(elharo): install these in third_party
+    compile files(ideaLib + 'jdom.jar')
 
-    testCompile group: 'junit', name: 'junit', version: '4.11+'
+    compile files('third_party/guava/guava-18.0.jar')
+
+    // todo(elharo): these two are only here for MockitoTestCase.
+    // Move that to testSrc and this becomes a testCompile dependency
+    compile files(ideaLib + 'junit-4.11.jar')
```
Comment (patflynn) [inline common-lib/build.gradle]: should we group all the ideaLIb imports together?

Comment (elharo) [inline common-lib/build.gradle]: This eventually won't be an INtelliJ dependency (+ more work needs to be done on this one)

**Comment by patflynn** [comment]
LGTM. Thanks for doing this.

**Pull request merged**